pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -745,6 +741,22 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo, provisionC
 	return deployment, nil
 }
 
+// applyOSDPlacement apply spec.placement.all if OnlyApplyOSDPlacement set to false
+func (c *Cluster) applyOSDPlacement(d *v1.PodSpec) {
```

Comment:
Since we couldn't pass the other arg, seems like we need a different method name such as:
```suggestion
func (c *Cluster) applyAllPlacementIfNeeded(d *v1.PodSpec) {
```

Reply:
done